config/environments/development.rb

```diff
@@ -38,6 +41,28 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  #
+  # Switch User
+  #
+  # Skip authentication for SkipUserController: http://stackoverflow.com/a/16623868/6763239
+  config.to_prepare do
+    SwitchUserController.skip_before_filter :authenticate_user!
+  end
```

Comment:
To enable `switch_user` (see also block below). Notice, again, `development` only.